src/serena/util/shell.py

```diff
@@ -1,9 +1,17 @@
 import os
 import platform
 import subprocess
+from typing import Any
 
 from pydantic import BaseModel
 
+# Import CREATE_NO_WINDOW safely across platforms
+try:
+    from subprocess import CREATE_NO_WINDOW
+except ImportError:
+    # Not available on non-Windows systems
+    CREATE_NO_WINDOW = 0x08000000
+
```

Comment:
This pattern doesn't make sense. The flag is only used for Windows, where it can also be imported. So it should be imported locally (in the Windows case) instead.